src/logic/data-source/DataSourceCommon.ts

```diff
@@ -0,0 +1,39 @@
+import FAElement from '../functional-annotations/FAElement';
+import { ProcessedPeptideContainer } from '../data-management/ProcessedPeptideContainer';
+import { Ontologies } from '../data-management/ontology/Ontologies';
+
+export namespace DataSourceCommon
+{
+    export function getFASummary(faElement: FAElement, processedPeptideContainer: ProcessedPeptideContainer = undefined): string[][]
```

Comment:
This function should normally be part of an abstract "FADataSource" class in the FA DataSource hierarchy, but this would require restructuring of the CountTables and DataSources which is out of the scope of this pull request.